doc/source/whatsnew/v0.21.1.txt

```diff
@@ -7,6 +7,39 @@ This is a minor release from 0.21.1 and includes a number of deprecations, new
 features, enhancements, and performance improvements along with a large number
 of bug fixes. We recommend that all users upgrade to this version.
 
+.. _whatsnew_0211.special:
+
+Restore Matplotlib datetime Converter Registration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandas implements some matplotlib converters for nicely formatting the axis
+labels on plots with ``datetime`` or ``Period`` values. Prior to pandas 0.21.0,
+these were implicitly registered with matplotlib, as a side effect of
+``import pandas``. In pandas 0.21.0, we required users to explicitly register
+the converter.
+
+.. code-block:: python
+
+   >>> from pandas.plotting import register_matplotlib_converters
+   >>> register_matplotlib_converters()
```

Comment:
I am not sure if I would show that here. Because people who glance over it might think they need to add that to their code, which is not the case.